src/main/rx/crsf.c

```diff
@@ -359,15 +360,32 @@ STATIC_UNIT_TESTED uint8_t crsfFrameStatus(rxRuntimeState_t *rxRuntimeState)
 STATIC_UNIT_TESTED float crsfReadRawRC(const rxRuntimeState_t *rxRuntimeState, uint8_t chan)
 {
     UNUSED(rxRuntimeState);
-    /* conversion from RC value to PWM
+    /* conversion from RC value to PWM for standard frame data 
      *       RC     PWM
      * min  172 ->  988us
      * mid  992 -> 1500us
      * max 1811 -> 2012us
      * scale factor = (2012-988) / (1811-172) = 0.62477120195241
      * offset = 988 - 172 * 0.62477120195241 = 880.53935326418548
      */
-    return (0.62477120195241f * (float)crsfChannelData[chan]) + 881;
+
+    /*    * new conversion from RC value to PWM for full range data
+     *       RC     PWM
+     * min  0    -> 988us
+     *      24   -> 1000us
+     * mid  1024 -> 1500us
+     *      2024 -> 2000us
+     * max  2048 -> 2012us
+     */
```

Comment:
```suggestion
    scale factor = (2012-988)  (2047-0) = 0.50024425989252564728871519296532
    offset =  988 - 0 * 0.50024425989252564728871519296532 = 881
     */
```
Question: I know it was already in the code but why is the range using `988 - 2012` instead of `988 - 2011`?

Reply:
You're right, actually shouldn't it be just (2012-988)  / (2048) = 0.5 ?

Reply:
Yes interesting how this compares to dshot resolution 48 - 2047 (1999 steps using float) to use with RC pwm of 1000 - 2000 matching RC value 24 - 2024